uiowa/uiowa
# [Contact] URL alias path

# How to test

1. Sync registrar
2. Create a piece of "Contact" content
3. Observe that its automatically generated alias is at `contact/[node:title]`
